SnoopCompileBot/src/SnoopCompileBot.jl

```diff
@@ -4,6 +4,7 @@ export BotConfig, snoop_bot, snoop_bench
 
 using Core: MethodInstance, CodeInfo
 using YAML
+using SnoopCompileCore   # needed for the [`@snoopi`](@ref) doc links
```

Comment:
Do we really need to do this? 🤔 I prefer to use full links instead of loading the package.
```suggestion
```

Reply:
I see your point, given that you do everything in external processes this is not ideal and should probably be deleted. But the manual cross-link is not as robust. Suppose someone makes some local changes and builds the docs locally. Then won't cross-links refer, not to their local copy of the documentation, but to the global one? That's not good.